webapp/src/cmds.ts

```diff
@@ -111,8 +111,18 @@ function localhostDeployCoreAsync(resp: pxtc.CompileResult): Promise<void> {
         url: "http://localhost:3232/api/deploy",
         headers: { "Authorization": Cloud.localToken },
         method: "POST",
-        data: resp
-    }).then(r => { });
+        data: resp,
+        allowHttpErrors: true
+    }).then(r => {
+        if (r.statusCode >= 400) {
+            if (r.text) {
+                core.errorNotification(r.text);
+            } else {
+                core.errorNotification(lf("Error during deployment"));
```

Comment:
The local server should notify that no boards were connected and ask the user to connect their micro:bit.